runtime/src/launch/slurm-gasnetrun_common/slurm-gasnetrun_common.h

```diff
@@ -346,6 +357,15 @@ int chpl_launch_handle_arg(int argc, char* argv[], int argNum,
     return 1;
   }
 
+  // handle --nodelist <nodelist> or --nodelist=<nodelist>
+  if (!strcmp(argv[argNum], CHPL_NODELIST_FLAG)) {
+    nodelist = argv[argNum+1];
+    return 2;
+  } else if (!strncmp(argv[argNum], CHPL_NODELIST_FLAG"=", strlen(CHPL_NODELIST_FLAG))) {
```

Comment:
Should the last arg be `strlen(CHPL_NODELIST_FLAG)+1`? As it stands the `strncmp()` call is not comparing against the `"="` that is appended to `CHPL_NODELIST_FLAG`.